src/main/steps/applicant2/review-your-response-to-the-courts-feedback/post.test.ts

```diff
@@ -0,0 +1,51 @@
+import { mockRequest } from '../../../../test/unit/utils/mockRequest';
```

Comment:
It would be good to add some functional tests that cover the RFI journey if possible. We could add them while the ticket is in QA though, or in another ticket before release if you'd prefer not to do them as part of this ticket

Reply:
Agreed.  I'd actually intended to add them, but got distracted with the additional tickets.

Given this PR is already quite large I'm wondering if they would be better off on their own.

Reply:
As discussed on slack, will handle the tests separately.